src/views/Account/Account.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import Layout from '../../components/Layout'
+import Alert from '../../components/Alert'
+import { withRouter } from 'react-router-dom'
+import auth from '../../api/auth';
+import { Link } from 'react-router-dom';
+import Skills from "../../components/Skill/Skills";
+import user from "../../api/user";
+
+export default withRouter(class Account extends Component {
+  state = {
+    id: (this.props.match.params.id),
+    error: null
+  }
+
+  /**
+  * Component setup
+  */
+  async componentDidMount() {
+    await this.getUser()
+  }
+
+  /**
+   * Get current authenticated user
+   * @return {Promise}
+   */
+  getUser = async () => {
+    if (this.state.id !== undefined) {
+      await this.setState(await user.get(this.state.id))
+    } else {
+      await this.setState(await user.get(auth.id()))
+    }
+  }
+
+  /**
+   * Creates the account view
+   * @return {JSX} View
+   */
+  render() {
+    let workerSpecifice;
+    if (this.state !== null && this.state.discriminator === 'Worker') {
+      workerSpecifice =
+        <div>
+          <Skills
+            workerId={this.state.id}
+          />
+        </div>
+    }
+
+    let editProfileButton
+    if (auth.id() === Number(this.state.id)) {
+      editProfileButton =
+        <Link to='/changeAccount'>
```

Comment:
Wrong route.

Reply:
Also wrap var assignment in ()

Reply:
Maybe move it to another render function to simplify the base render method. Makes it easier to read.